include/bluetooth/addr.h

```diff
@@ -87,6 +133,15 @@ static inline bool bt_addr_le_is_rpa(const bt_addr_le_t *addr)
 	return BT_ADDR_IS_RPA(&addr->a);
 }
 
+/** @brief Check if a Bluetooth LE address is valid identity address.
+ *
```

Comment:
There's some incoherence with
```
/** @brief 
```
and
```
/**
 * @brief
```

in this PR. 

I'm no longer sure what the ideal way is, since `bluetooth.h` was refactored in that regards lately, but some of the other Bluetooth includes weren't...